### ISSUE TITLE
Incorrect placement; use within Bootstrap classes

### ISSUE THREAD

**Issue by boozer87** [uvarov-frontend/vanilla-calendar#134]
Many thanks for the excellent plugin. 

However, due to improper placement within Bootstrap classes (5.1.3), the calendar will always appear at the top of the page.

```
<div class="row pt-4 align-items-center">
      <div class="col-lg-3">
          <label for="dep_date" class="text-16 lh-1 fw-500 mb-2">Departure Date</label>
          <div class="form-input">
              <input type="text" name="dep_date" id="dep_date">
          </div>
      </div>
  </div>
```
```
document.querySelector('#dep_date').style.display = "flex";
    const options = {
      input: true,
      actions: {
        changeToInput(e, calendar, dates, time, hours, minutes, keeping) {
          if (dates[0]) {
            calendar.HTMLInputElement.value = dates[0];
            calendar.hide();
          } else {
            calendar.HTMLInputElement.value = "";
          }
        }
      }
    };
    const calendarInput = new VanillaCalendar("#dep_date", options);
    calendarInput.init();
```
![image](https://github.com/uvarov-frontend/vanilla-calendar/assets/10062240/848dacf3-f3f1-4952-8021-3c2bdfe6c66c)

Whenever we remove bootstrap classes then it displays properly. It only works properly when we define it as input on its own.

Are you able to verify it?

Thank you for your assistance.
Greetings.

**Comment by boozer87** [comment]
any news?

**Comment by uvarov-frontend** [comment]
Hi. Make an example in sandbox, please, I'll take a look soon.

**Comment by boozer87** [comment]
Hi again, It's working on sandbox : ) 

https://rxmdrt.csb.app/

Maybe my other CSS files are affected. I'm sorry for this.

**Comment by uvarov-frontend** [comment]
You didn't import `bootstrap` in your example.

**Comment by boozer87** [comment]
Sorry, updateded and import bootstrap. please check again.

https://rxmdrt.csb.app/

**Comment by uvarov-frontend** [comment]
Fixed in v2.6.9, thanks for reporting the issue.

**Comment by boozer87** [comment]
Thanks for your time and effort. You're the greatest. Cheers!

**Comment by uvarov-frontend** [comment]
There is another problem with positioning after scrolling the page.
I'm fixing it.

**Comment by boozer87** [comment]
Thank you very much, but testing with local files on the most recent Release v2.6.9 revealed the same issue. Is the problem with page scrolling? Also, does this link include all releases: http://cdn.jsdelivr.net/npm/@uvarov.frontend/vanilla-calendar/build/vanilla-calendar.min.js?

**Comment by uvarov-frontend** [comment]
I uninstalled release 2.6.9 because it contained bugs that was just released. I will release a new release within an hour.

**Comment by boozer87** [comment]
Thank you, I look forward to your response, and I have attached some additional images for the test website.

here is bottom of the page,
![image](https://github.com/uvarov-frontend/vanilla-calendar/assets/10062240/2ec74636-ce00-4517-bca6-f11049acd1d1)

![image](https://github.com/uvarov-frontend/vanilla-calendar/assets/10062240/c5c10590-857d-4aa3-84b6-079fd4aeda60)
calendar input here.

**Comment by uvarov-frontend** [comment]
Fixed in v2.7.0

**Comment by boozer87** [comment]
Thank you very much! Works perfect.

![image](https://github.com/uvarov-frontend/vanilla-calendar/assets/10062240/69a5c34b-a3f8-4dba-ad71-863591e5b6db)

Cheers.
